File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/EnrollmentResponse.ts
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type EnrollmentResponse = DomainResource & {
    status?: string;
    created?: string;
    outcome?: string;
    request?: Reference;
    identifier?: Identifier[];
    disposition?: string;
    organization?: Reference;
    requestProvider?: Reference;
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/EnrollmentResponse.py
from typing import Optional, List
from base import DomainResource
from base import Reference
from base import Identifier

class EnrollmentResponse(DomainResource):
    status: Optional[str] = None
    created: Optional[str] = None
    outcome: Optional[str] = None
    request: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = None
    disposition: Optional[str] = None
    organization: Optional[Reference] = None
    request_provider: Optional[Reference] = None